decomfl_main.py

```diff
@@ -9,31 +9,57 @@
 from byzantine import aggregation as byz_agg
 from byzantine import attack as byz_attack
 from cezo_fl.client import ResetClient
-from cezo_fl.fl_helpers import get_client_name
+from cezo_fl.fl_helpers import get_client_name, get_server_name
 from cezo_fl.server import CeZO_Server
-from cezo_fl.util import model_helpers, prepare_settings
-
-from config import get_args_str, get_params
-from preprocess import preprocess
+from cezo_fl.util import model_helpers
+from experiment_helper import prepare_settings
+from experiment_helper.cli_parser import (
+    GeneralSetting,
+    DeviceSetting,
+    DataSetting,
+    ModelSetting,
+    OptimizerSetting,
+    FederatedLearningSetting,
+    RGESetting,
+    ByzantineSetting,
+)
+from experiment_helper.device import use_device
+from experiment_helper.data import get_dataloaders
+
+
+class CliSetting(
+    GeneralSetting,
+    DeviceSetting,
+    DataSetting,
+    ModelSetting,
+    OptimizerSetting,
+    FederatedLearningSetting,
+    RGESetting,
+    ByzantineSetting,
+):
+    pass
```

Comment:
add comment about this class since this is not obvious at the first look

Reply:
will add in the end after we resolve the other comments